x-pack/plugins/enterprise_search/public/applications/app_search/components/engine/types.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { IApiToken } from '../credentials/types';
+import { Schema, SchemaConflicts, IndexingStatus } from '../schema/types';
```

Comment:
I started removing the I/E/T prefixes from our types per our discussion in Slack today.